src/pages/EditMonitor.vue

```diff
@@ -285,6 +325,18 @@ export default {
 
         pushURL() {
             return this.$root.baseURL + "/api/push/" + this.monitor.pushToken + "?msg=OK&ping=";
+        },
+
+        bodyPlaceholder() {
+            return `{
+    "id": 124357,
+    "username": "admin",
+    "password": "myAdminPassword"
+}`;
+        },
```

Comment:
```suggestion
        bodyPlaceholder() {
            return `{
                "id": 124357,
                "username": "admin",
                "password": "myAdminPassword"
            }`;
        },
```

Reply:
Won't the textarea then show this:
```
{
                "id": 124357,
                "username": "admin",
                "password": "myAdminPassword"
            }
```

instead of this?:
```
{
    "id": 124357,
    "username": "admin",
    "password": "myAdminPassword"
}
```

Reply:
Maybe

Reply:
yes:
![image](https://user-images.githubusercontent.com/1710840/135976627-c2626496-0abc-46c3-8a9d-e1242afe70b6.png)